src/VecSim/algorithms/hnsw/hnsw_factory.cpp

```diff
@@ -1,11 +1,10 @@
 #include "VecSim/algorithms/hnsw/hnsw_factory.h"
-#include "VecSim/algorithms/hnsw/hnsw_wrapper.h"
-#include "VecSim/algorithms/hnsw/hnswlib.h" //linklistsizeint
-#include "VecSim/vec_sim_common.h"          // labelType
+#include "VecSim/algorithms/hnsw/hnsw.h" //linklistsizeint
```

Comment:
I don't like these comments next to the includes...

Reply:
@DvirDukhan WDYT? I will align the code when Meirav's PRs will get to the main branch.

Reply:
I agree